be/src/olap/tablet_schema.cpp

```diff
@@ -246,6 +256,7 @@ uint32_t TabletColumn::get_field_length_by_type(TPrimitiveType::type type, uint3
         return 4;
     case TPrimitiveType::DOUBLE:
         return 8;
+    case TPrimitiveType::QUANTILE_STATE:
```

Comment:
Why ```QUANTILE_STATE``` share the same type length with ```Object Type```?
Is QUANTILE variable length type or fixed length?

Reply:
QUANTILE_STATE is variable length type, which is same to BITMAP.

It only store `ptr`(8 byte) and `length`(8 byte) in the TupleRow.

Reply:
QUANTILE_STATE is variable length type, which is same to BITMAP.

It only store ptr(8 byte) and length(8 byte) in the TupleRow.